.github/workflows/docker-release.yml

```
@@ -0,0 +1,48 @@
+name: Daily build
+on:
+  push:
+    branches:
+      - '**'
+  schedule:
+    - cron: "0 1 * * *"
+
```

Let's add   `workflow_dispatch: {}` so you can also start the release manually if needed
```suggestion
  workflow_dispatch: {}
```